src/app/modules/pictureapp-explore/components/edit-userProfile/edit-userProfile.component.html

```diff
@@ -0,0 +1,22 @@
+
+<div layout="row" class="edit-userPfrofile-background-image">
+    <div flex class="edit-userProfile-backgound-image-basePosition"></div>
+    <div class="edit-userProfile-mainPhoto-flex-container-element-image">
+      <img src="{{mainPhoto?.url}}" class="edit-userProfile-mainPhoto-image"/>
```

Comment:
Is it works same like operator in C# `{{mainPhoto?.url}}`? If so, why do we assume that mainPhoto could be null?

Reply:
the ? opertor use  only for safe view render